### ISSUE TITLE
animmex.com / animmex.net

### ISSUE THREAD

**Issue by sdfrhuhs** [reek/anti-adblock-killer#1718]
[https://www.animmex.com](url)
[https://www.animmex.net](url)
specific video example-- [test](https://www.animmex.net/video/13413/the-simpsons-s27e12-much-apu-about-something)

screenshot- [http://imgur.com/qd3pkEO](url)

The problem occurs not on the main page, but only after a video is clicked.

Google Chrome
Tampermonkey
Adblock Plus/ (Adblock) tried with either or disabled
EasyList + AakList

I also run Referer Control, and this has also mysteriously stopped working. Was good for a few months, no work arounds. Now no matter what I do it doesn't work, and I get bounced to the disable adblock page. 

I know of this duplicate [https://github.com/reek/anti-adblock-killer/issues/1704](url), but the workaround does not work, and I believe the issue is that blockadblock.com has updated its script, so felt justified in opening a new issue.

**Comment by Ntweat** [comment]
FYI they are using a script which tracks EasyList.. It is called blockadblock.com

**Comment by jonasthambert** [comment]
I'm using uBlock + Ghostery. Not able to reproduce.

**Comment by maynak00** [comment]
**try with:**
`@@||animmex.net$generichide`

**Comment by reek** [comment]
Ok, I'll kill it, thanks

**Comment by reek** [comment]
Hi,<br><br>Anti-AdBlock Killer has been updated ! <a href="https://github.com/reek/anti-adblock-killer#changelog">Changelog</a><br><br>Step 1: <a href="https://raw.githubusercontent.com/reek/anti-adblock-killer/master/anti-adblock-killer.user.js">Update</a> : UserScript<br>Step 2: <a href="http://reek.github.io/anti-adblock-killer/#filterlist">Update</a> : FilterList<br><br>Do not forget to donate, review, share, follow and add to favorites.<br><br>See you<br>Reek

https://github.com/reek/anti-adblock-killer/
https://twitter.com/antiadbkiller
